pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools >= 66.1.0, <= 72.1.0"]  # lower bound to support controller Python versions, upper bound for latest version tested at release
+requires = ["setuptools >= 66.1.0, <= 72.1.0", "wheel == 0.45.1"]  # lower bound to support controller Python versions, upper bound for latest version tested at release
```

Comment:
NOTE for history: v70.1.0 is when `wheel` became a part of the `setuptools` project. I imagine when the minimum bound is higher than that, the `wheel` pinning hack can be dropped.

https://setuptools.pypa.io/en/latest/history.html#v70-1-0